optional_plugins/varianter_yaml_to_mux/avocado_varianter_yaml_to_mux/__init__.py

```diff
@@ -342,8 +335,6 @@ def configure(self, parser):
         """
         Configures "run" and "variants" subparsers
         """
-        if not MULTIPLEX_CAPABLE:
-            return
```

Comment:
Would you please split this little modification to a separate commit?

Reply:
This "little" modification means that all the tests in the current location will need an alternative check, that is, an "is plugin avaialable?" check.  I found it counter intuitive, but I've done so you can see what I mean clearly.